data_models/data_refinery_models/migrations/0002_auto_20170727_2100.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
Because I wanted to clear out the production DB, so I took the opportunity to clean up the migration files since I could just reapply them all from scratch. At this point, there is a lot of data in the database so I think this is the last time I can do this.